docs/source/basics.md

```diff
@@ -121,9 +121,17 @@ This can be saved into the value `first_val` with this response block:
 response:
   save:
     body:
-      fourth_val: thing.nested.0
+      first_val: thing.nested[0]
 ```
 
+The query should be defined as a JMES query (see http://jmespath.org/
+for more information). In the above example, this essentially performs
+the operation `json["thing"]["nested"][0]`. This can be used to perform
```

Comment:
I started drafting some changes to "In the above example, this essentially performs the operation `json["thing"]["nested"][0]`" but I think in reality it's probably best to leave it out, because it breaks the flow of the paragraph. Perhaps add a yaml comment with `# The python equivalent of this selector would be json["thing"]["nested"][0]`

Reply:
I think that sentence only makes sense of the context of it being a JMES path query though